### ISSUE TITLE
Reorder Make Offer Form Fields..??

### ISSUE THREAD

**Issue by angelleye** [angelleye/offers-for-woocommerce#279]
* https://wordpress.org/support/topic/reorder-fields-on-make-offer-form/

**Comment by angelleye** [comment]
@tejasm-itpath It looks like you simply moved the one field this particular user was asking for here..??  What we really should do here is make it so that people can arrange the fields in any order they want.

This goes back to #248 where we added the check boxes to allow people to set fields as required.  In that issue I made the following comment that I think you must have missed:

>I think what we'll probably do here is replace the current check boxes with some sort of a panel where you can enable pages and also drag and drop them to re-order them. Then we can also add required check box at that time.

So as that says, I think a drag-n-drop interface for the form field options would be good.  People could drag the fields to the order they want them to show up on the form, and also check boxes to make each required.

I don't want to delay this release, so we'll move this to the next update.
